src/components/backend-ai-app-launcher.ts

```diff
@@ -1265,6 +1269,39 @@ export default class BackendAiAppLauncher extends BackendAIPage {
     collapsibleArea.style.maxHeight = isFolded ? '100%': '83px';
     btn.textContent = isFolded ? _text('session.Readless') : _text('session.Readmore');
   }
+  /**
+   * Copy SSH Connection Example to Clipboard
+   */
+
+  _copySSHConnectionExample(divSelector) {
+    const divElement = this.shadowRoot?.querySelector(divSelector);
+    if (divElement) {
+      const textToCopy = divElement.textContent;
+
+      if (textToCopy.length === 0) {
+        this.notification.text = _text('emptyDivMessage');
+        this.notification.show();
+      } else {
+        if (navigator.clipboard !== undefined) { // for Chrome, Safari
+          navigator.clipboard.writeText(textToCopy).then(() => {
+            this.notification.text = _text('SSHConnectionExampleClipboardCopy');
```

Comment:
Refer to https://github.com/lablup/backend.ai-webui#localization, please update `<language>.json` file and apply it.